src/change_stream.ts

```diff
@@ -219,69 +310,63 @@ export class ChangeStream extends EventEmitter {
    * @function
    * @param {Writable} destination The destination for writing data
    * @param {object} [options] {@link https://nodejs.org/api/stream.html#stream_readable_pipe_destination_options|Pipe options}
-   * @returns {null}
    */
-  pipe(destination: any, options?: object): null {
+  pipe(destination: Writable, options?: { end?: boolean }): Writable {
```

Comment:
I did this because its inlined in the node types, but I pulled it out, not sure if they have a set of rules over there

```ts
pipe<T extends NodeJS.WritableStream>(destination: T, options?: { end?: boolean; }): T;
```